os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -715,12 +836,25 @@ static void up_dumpstate(void)
 
 	up_registerdump();
 
+#ifdef CONFIG_FRAME_POINTER
+	/* Dump the stack */
+	lldbg("*******************************************\n");
+	lldbg("Call stack of aborted task:\n");
+	lldbg("*******************************************\n");
 	dump_stack();
-
+#endif
 	/* Dump the state of all tasks (if available) */
 
+	lldbg("*******************************************\n");
+	lldbg("List of all tasks in the system:\n");
+	lldbg("*******************************************\n");
 	up_showtasks();
 
+#ifdef CONFIG_FRAME_POINTER
+	/* Display the call stack of all tasks */
+	dump_all_stack();
```

Comment:
aborted task will be printed twice by dump_stack and dump_all_stack. Is it intended?

Reply:
Yes. It's intended.  Main objective of this API is to print the call stack of all the tasks/threads in the system.  Since aborted task is also one of the available task in g_readytorun list, It displays twice.